testcases/kernel/syscalls/fallocate/fallocate03.c

```diff
@@ -162,9 +173,9 @@ void setup(void)
 	/* Create temporary directories */
 	TEST_PAUSE;
 
-	tst_tmpdir();
+	mkdir(tempdir, 0777);
 
-	sprintf(fname, "tfile_sparse_%d", getpid());
+        sprintf(fname, "%s/tfile_sparse_%d", tempdir, getpid());
```

Comment:
Indentation. Also as we discussed use hard coded file name instead of creating with sprintf, we are using fresh image to run each LTP test so it is OK to use same file name.